.github/workflows/lint.yml

```diff
@@ -1,7 +1,7 @@
 name: Lint
 
 on:
-  pull_request:
+  # pull_request:
```

Comment:
Don't forget to uncomment this line after removing draft status on pull request (comment that was added for quicker test)